frontend-react/.env.production

```diff
@@ -1,8 +1,17 @@
-; production overrides .env values
-REACT_APP_BASE_URL=https://reportstream.cdc.gov
+; this is set by env var being set `CLIENT_ENV: production` in the github action
+REACT_APP_TITLE=CDC Prime ReportStream
+REACT_APP_DESCRIPTION=ReportStream Dashboard Application
+REACT_APP_ICON=https://reportstream.cdc.gov/assets/cdc-logo.svg
+
+REACT_APP_BASE_URL=https://prime.cdc.gov
```

Comment:
This could actually stay `https://reportstream.cdc.gov`, but `AuthResource.getBaseUrl()` should change to return `REACT_APP_BACKEND_URL`. Returning `REACT_APP_BASE_URL` as we do now suggests we're trying to return the front-end URL rather than the API URL.

Reply:
OK. Agreed. 

I'm going to remove the `getBaseUrl` function and refactor.